docs/faq.rst

```diff
@@ -353,6 +353,15 @@ down and up again. A container restart is not sufficient.
 
 *Issue reference:* `615`_.
 
+SMTP Banner from overrides/postfix.cf is ignored
+````````````````````````````````````````````````
+
+Any mail related connection is proxied by nginx. Therefore the SMTP Banner is also set by nginx. Overwriting in overrides/postfix.cf does not apply. If you want to change the hostname in the SMTP Banner greeting, you must change the first entry of HOSTNAMES in mailu.env.
```

Comment:
```suggestion
Any mail related connection is proxied by nginx. Therefore the SMTP Banner is also set by nginx. Overwriting in overrides/postfix.cf does not apply.
```

I think it would be better not to include anything about changing hostnames here, since that could lead to more errors when people try weird things without asking for help first :D